plone/app/event/dx/behaviors.py

```diff
@@ -1,6 +1,7 @@
 """Behaviors to enable calendarish event extension to dexterity content types.
 """
 
+from plone import api
```

Comment:
core packages do not use `plone.api`... use 
```
from zope.component.hooks import getSite

portal = getSite()
```

Reply:
ok, thanks for hint. the import is removed.